src/util/libm.rs

```diff
@@ -129,7 +119,7 @@ impl Float for f64 {
             return x;
         }
         unsafe {
```

Comment:
I think you forgot to remove this `unsafe`?

Reply:
Yep. Fixed.